Client/Frontend/DefaultBrowserOnboarding/DefaultBrowserOnboardingViewController.swift

```diff
@@ -102,7 +102,7 @@ class DefaultBrowserOnboardingViewController: UIViewController, OnViewDismissabl
         label.accessibilityIdentifier = AccessibilityIdentifiers.FirefoxHomepage.HomeTabBanner.descriptionLabel3
     }
 
-    private lazy var goToSettingsButton: ResizableButton = .build { button in
+    private lazy var goToSettingsButton: PrimaryButton = .build { button in
         button.layer.cornerRadius = UX.ctaButtonCornerRadius
```

Comment:
If we use `PrimaryButton` then we can avoid resetting all the properties here, and use the viewModel to set the accessibilityIdentifier and title of that button. See https://github.com/mozilla-mobile/firefox-ios/pull/16417 for an example of that

Reply:
Do we need to use PrimaryButton or PrimaryRounded Button? Can you give clarity on that, I'm a bit confused.